jobs/build/advisories/Jenkinsfile

```diff
@@ -184,6 +197,32 @@ node {
                     """
                 }
             }
+            stage("sending email to request Live IDs") {
+                if (params.REQUEST_LIVE_IDs) {
+                    def live_id_content = """
+                        Hello docs team,I'd like to request Live IDs for our ${params.VERSION}.z advisories:\n
+                        RPM: https://errata.devel.redhat.com/advisory/${rpm_advisory_id}\n
+                        Image: https://errata.devel.redhat.com/advisory/${image_advisory_id}\n
+                    """
+                    if (params.VERSION.startsWith("4")) {
+                        live_id_content += """
+                        Extras: https://errata.devel.redhat.com/advisory/${extras_advisory_id}\n
+                        OLM Operators metadata: https://errata.devel.redhat.com/advisory/${metadata_advisory_id}\n
+                        """
+                    }
```

Comment:
```suggestion
                    def live_id_content = """
                        Hello docs team,I'd like to request Live IDs for our ${params.VERSION}.z advisories:\n
                        Image: https://errata.devel.redhat.com/advisory/${image_advisory_id}\n
                    """

Reply:
Well, uh....
`Hello docs team, ART would like to request a Live ID for our ${params.VERSION}.z advisory:\n`

Reply:
Sure, ART is better, I just bumped from jira template